flaml/automl.py

```diff
@@ -2502,6 +2550,7 @@ def _search_sequential(self):
                         if isinstance(search_state.init_config, list)
                         else [search_state.init_config]
                     )
+
```

Comment:
Isn't the change for optuna applicable to this function _search_sequential() as well?